hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/hdfs/inode/InodeFacade.java

```diff
@@ -152,6 +152,22 @@ public Inode findById(Long id) {
     }
   }
   
+  /**
+   * find inode by id
+   * <p/>
+   * @param id
+   * @return
+   */
+  public List<Inode> findByIdList(List<Long> ids) {
+    TypedQuery<Inode> q = em.createNamedQuery("Inode.findByIdInList", Inode.class);
+    q.setParameter("idList", ids);
+    try {
+      return q.getResultList();
+    } catch (NoResultException e) {
```

Comment:
`getResultList` does not throw a `NoResultException` : https://docs.oracle.com/javaee/6/api/javax/persistence/Query.html#getResultList()

Reply:
Just return an empty list instead of dealing with null in the caller

Reply:
I do a null check and return a new empty list since apparently depending on the implementation, the getResultList might return null sometime:
https://stackoverflow.com/questions/1115480/can-javax-persistence-query-getresultlist-return-null